centaur/src/main/resources/standardTestCases/output_filename_interpolation.test

```diff
@@ -15,5 +15,5 @@ metadata {
 
 fileSystemCheck: "gcs"
 outputExpectations {
-  "gs://cloud-cromwell-dev/output_filename_interpolation_wf/<<UUID>>/call-output_filename_interpolation/Upside": 1
+  "gs://cloud-cromwell-dev/output_filename_interpolation_wf/<<UUID>>/call-output_filename_interpolation/baz": 1
```

Comment:
I believe `baz` would actually contain `gs://[...]/bar`. And I also believe that the `bar` file won't exist.